examples/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM gradle:jdk14 AS builder
 COPY . /home/gradle/src
 WORKDIR /home/gradle/src/situp-core
-RUN gradle jar --daemon
+RUN gradle cleanjar --daemon
```

Comment:
This does not work on my laptop.

Reply:
My bad, this was a typo. Fixing in the next commit. `RUN gradle clean jar --daemon`